debezium-quarkus-outbox/runtime/pom.xml

```diff
@@ -23,6 +23,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jackson</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I still wished this was an optional feature. It add a (small) overhead after all, and someone without the tracing infrastructure in place might not be willing to pay that overhead.

Reply:
Ok, I think we can track this separately, if needed. @jpechane, could you log an issue for it? It's ok to require it for the time being in the Quarkus extension, but it should not be needed for the routing SMT (as the update would be rather disruptive then).

Reply:
@gunnarmorling https://issues.redhat.com/browse/DBZ-2834